Utilities/QC/QCMerger/src/runMerger.cxx

```diff
@@ -97,7 +97,6 @@ int main(int argc, char** argv)
   bpo::notify(vm);
 
   MergerDevice mergerDevice(unique_ptr<Merger>(new Merger(NUMBER_OF_QC_OBJECTS_FOR_COMPLETE_DATA)), MERGER_DEVICE_ID);
-  mergerDevice.CatchSignals();
```

Comment:
Signals catching is done in the supplied plugin. These QC devices should switch to using DeviceRunner, that includes all the plugins/configuration facilities.